src/components/Modal/ModalContent/MovieForm/MovieForm.jsx

```diff
@@ -1,78 +1,204 @@
-import React, {useState, useContext} from "react";
+import React from "react";
 import Button from "../../../Button/Button";
 import FormRow from "../../../Forms/FormRow";
-import ModalContext from "../../../../ModalContext";
+import {connect} from "react-redux";
+import { DateTime } from "luxon";
+import {Field, FieldArray, Formik} from "formik";
+import {editMovie, addMovie} from "../../../../redux/movie/action-creators";
 
-function useResetInput(initialValue) {
-    const[value, setValue] = useState(initialValue);
+const MovieForm = (props) => {
+    const selectOptions = ["All", "Documentary", "Comedy", "Horror", "Crime"];
```

Comment:
I think this const can be moved out of component because it does not depend on any props

Reply:
fixed